### PR TITLE
adding 4.13 tests and removing 4.9

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>

### CODE REVIEW

File: ci-operator/jobs/redhat-developer/odo/redhat-developer-odo-main-periodics.yaml [hunk 1]
```diff
@@ -340,18 +340,10 @@ periodics:
   labels:
     ci-operator.openshift.io/cloud: aws
     ci-operator.openshift.io/cloud-cluster-profile: aws
-    ci-operator.openshift.io/variant: v4.9
+    ci-operator.openshift.io/variant: v4.13
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-redhat-developer-odo-main-v4.9-integration-e2e-periodic
-  reporter_config:
```
Comment (kadel) [inline ci-operator/jobs/redhat-developer/odo/redhat-developer-odo-main-periodics.yaml:347]: It looks like there is no reporter_config for 4.13.  Does this mean that we won't get slack notifications?

**Comment by anandrkskd** [comment]
/pj-rehearse periodic-ci-redhat-developer-odo-main-v4.13-integration-e2e-periodic

**Comment by anandrkskd** [comment]
/pj-rehearse periodic-ci-redhat-developer-odo-main-v4.13-integration-e2e-periodic

File: ci-operator/jobs/redhat-developer/odo/redhat-developer-odo-main-periodics.yaml (hunk 1 above)
Comment (anandrkskd) [reply ci-operator/jobs/redhat-developer/odo/redhat-developer-odo-main-periodics.yaml:359]: Updated!

**Comment by anandrkskd** [comment]
Can we merge this PR?
cc_ @feloy  @rm3l @valaparthvi

**Review by feloy — APPROVED** [review]

**Pull request merged**